org.onionware.onionmedia.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.onionware.onionmedia
```

Comment:
`onionware.org` doesn't exists. The app-id doesn't meet the requirements.

Reply:
Do I have to buy a domain or is there another way to find a good app ID?

Reply:
See https://docs.flatpak.org/en/latest/conventions.html#application-ids how to use github.

Reply:
Ok, so I can use io.github.onionware.onionmedia?

Reply:
`io.github.onionware_github.onionmedia`